cinder/templates/volumes/volumes.yaml

```diff
@@ -0,0 +1,21 @@
+#TODO For vmware we should use the hypervisor definition to define volume nodes to avoid repeating config - consider moving hypervisor, volume, share and network hardware definitions into global section.
+{{- $context := . -}}
```

Comment:
This is not really needed. The inital scope of "." can be accessed in a range loop by using "$." instead.